### PR TITLE
W7 - TienHuynh3 - UI Test

### PR DESCRIPTION
**Pull request by at-tienhuynh3**
#### Related issues (Optional)
Task #84 

#### Description (Required)
- [x] UI Test

### CODE REVIEW

File: app/src/androidTestAt-tienhuynhDebug/java/UiTest.java [hunk 1]
```diff
@@ -0,0 +1,253 @@
+import android.support.test.espresso.Espresso;
```
Comment (at-dev-support-01) [inline app/src/androidTestAt-tienhuynhDebug/java/UiTest.java]: WARNING: [TooManyStaticImports] Too many static imports may lead to messy code
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/imports.html#TooManyStaticImports

File: app/src/androidTestAt-tienhuynhDebug/java/UiTest.java [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.not;
+import static org.hamcrest.core.StringStartsWith.startsWith;
+
+/**
+ * Created by TienHuynh on 04/01/2018.
+ * AsianTech Co., Ltd
+ */
+@RunWith(AndroidJUnit4.class)
+public class UiTest {
```
Comment (at-dev-support-01) [inline app/src/androidTestAt-tienhuynhDebug/java/UiTest.java]: WARNING: [NoPackage] All classes and interfaces must belong to a named package
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#NoPackage

File: app/src/androidTestAt-tienhuynhDebug/java/UiTest.java [hunk 3]
```diff
@@ -0,0 +1,253 @@
...
+    }
+
+    /**
+     * check Toast Password Different UserName
+     */
+    @Test
+    public void checkToastPasswordDifferentUserName() {
+        // type text and press button.
+        Espresso.onView(withId(R.id.edtUserName))
+                .perform(typeText("TIEN271ABC"), closeSoftKeyboard());
```
Comment (at-dev-support-01) [inline app/src/androidTestAt-tienhuynhDebug/java/UiTest.java]: WARNING: [AvoidDuplicateLiterals] The String literal "TIEN271ABC" appears 8 times in this file; the first occurrence is on line 136
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/strings.html#AvoidDuplicateLiterals

File: app/src/androidTestAt-tienhuynhDebug/java/UiTest.java (hunk 1 above)
Comment (at-dev-support-01) [inline app/src/androidTestAt-tienhuynhDebug/java/UiTest.java]: WARNING: [TooManyStaticImports] Too many static imports may lead to messy code
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/imports.html#TooManyStaticImports

File: app/src/androidTestAt-tienhuynhDebug/java/UiTest.java (hunk 2 above)
Comment (at-dev-support-01) [inline app/src/androidTestAt-tienhuynhDebug/java/UiTest.java]: WARNING: [NoPackage] All classes and interfaces must belong to a named package
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#NoPackage

File: app/src/androidTestAt-tienhuynhDebug/java/UiTest.java (hunk 3 above)
Comment (at-dev-support-01) [inline app/src/androidTestAt-tienhuynhDebug/java/UiTest.java]: WARNING: [AvoidDuplicateLiterals] The String literal "TIEN271ABC" appears 8 times in this file; the first occurrence is on line 136
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/strings.html#AvoidDuplicateLiterals

Comment (at-tienhuynh3) [reply app/src/androidTestAt-tienhuynhDebug/java/UiTest.java]: i fixed it.

File: app/src/androidTestAt-tienhuynhDebug/java/UiTest.java (hunk 1 above)
Comment (at-dev-support-01) [inline app/src/androidTestAt-tienhuynhDebug/java/UiTest.java]: WARNING: [TooManyStaticImports] Too many static imports may lead to messy code
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/imports.html#TooManyStaticImports

File: app/src/androidTestAt-tienhuynhDebug/java/UiTest.java (hunk 2 above)
Comment (at-dev-support-01) [inline app/src/androidTestAt-tienhuynhDebug/java/UiTest.java]: WARNING: [NoPackage] All classes and interfaces must belong to a named package
https://pmd.github.io/pmd-5.6.1/pmd-java/rules/java/naming.html#NoPackage

**Comment by at-tienhuynh3** [comment]
@at-trungnguyen pls review for me, thank you <3

**Pull request merged**